src/main/java/org/jenkinsci/plugins/workflow/cps/Snippetizer.java

```diff
@@ -66,6 +66,7 @@
 import org.kohsuke.stapler.QueryParameter;
 import org.kohsuke.stapler.Stapler;
 import org.kohsuke.stapler.StaplerRequest;
+import org.springframework.util.ClassUtils;
```

Comment:
Mm, rather not use `springframework` please. Better just revert this last change.